Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/FunctionParametersNewlineConfiguration.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftLintCore
+
+@AutoApply
+struct FunctionParametersNewlineConfiguration: SeverityBasedRuleConfiguration {
+    typealias Parent = FunctionParametersNewlineRule
```

Comment:
Not sure if it makes sense. 
But we could also include a configuration so that the user could define how many params is needed before this rule is takes into account? Some people properly will allow `self.foo(bar: "bar", baz: "baz")` etc.